### ISSUE TITLE
Roam "Document“ style missing

### ISSUE THREAD

**Issue by tombarys** [roam-garden/i#19]
**Describe the bug**
It is not possible to publish blocks in Document mode (without bullets).

**To Reproduce**
1. Create a page in Roam with text content and set all (or some) blocks to Document mode
2. Plant a garden
3. See all text is bulleted

**Expected behavior**
Roam.Garden should strictly differentiate between block modes.

**Comment by Stvad** [comment]
Yeah, this information is missing from JSON export :(. So as long as I depend on it as source of truth, this will be challenging.  
One workaround is to remove bullets with CSS, though you won't be able to be very selective with that.

**Comment by tombarys** [comment]
Idea: what about adding a special tag for flagging a particular page as a Document-based on the first level? I guess it could be a workaround that should work for most situations when you want to simply share basic texts, poems, etc. Children of the doc-tagged page would be simple paragraphs (non-bullets), but children of this children could behave like bullets again.

**Comment by Stvad** [comment]
That can be an option. I'm thinking about implementing convention of inferring the class for the block based on the tags inside similar to how Roam does, and this can be one of the use-cases. Need to do some refactoring first though. Btw you may find #17 interesting as it implements what you described for the whole Garden

**Comment by tombarys** [comment]
Great. Thanks!

**Comment by Stvad** [comment]
#26 would be an enabled for something like that

**Comment by JayFarei** [comment]
> Idea: what about adding a special tag for flagging a particular page as a Document-based on the first level? I guess it could be a workaround that should work for most situations when you want to simply share basic texts, poems, etc. Children of the doc-tagged page would be simple paragraphs (non-bullets), but children of this children could behave like bullets again.

In roam I use a similar technique => tag= .flattener

.flattener div.rm-block-children  { margin-left:0!important;}
.flattener div.rm-multibar {border-right: 0px}

so long as we have classes in roam.garden to match that should do it?
(although I struggle with empty blocks)

**Comment by Stvad** [comment]
Sounds reasonable. Also with #28 - I actually can get the "document" information as I do export manually vs relying on the default JSON export process. It'd only work when you use the plugin though 🤔
